mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -57,7 +57,16 @@ def get_supported_targets():
         Load target mapping information
         """
         mbeds = mbed_lstools.create()
-        return sorted(set(mbeds.manufacture_ids.values()))
+        try:
+            # this works for <v1.3.0
+            list_supported_targets = sorted(set(mbeds.manufacture_ids.values()))
+        except AttributeError:
+            try:
+                # this should works for >=v1.3.0
+                list_supported_targets = sorted(set(list(mbeds.plat_db._dbs.values())[0].values()))
```

Comment:
Please don't use private variables (prefixed with a `_`). Instead you can use `plat_db.items()`

Reply:
is this only way to get names for now until we have better API for this?

Reply:
Yes.